### PR TITLE
DOCS-8087: Support matrix for CSM Agentless Scanning

### PR DESCRIPTION
**Pull request by drichards-87**
### What does this PR do? What is the motivation?

DOCS-8087. Support matrix for CSM Agentless Scanning

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/security/cloud_security_management/agentless_scanning/compatibility.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+| Ubuntu                   | All versions supported by Canonical                 | apt/dpkg         | [https://ubuntu.com/security/cve][15]                                                                     |
+
+## Software Composition Analysis
+
+The following languages are supported for Software Composition Analysis scans on containers and Lambda instances:
+
+| Language | Supported Package Manager | Supported Files                                                      |
+|----------|---------------------------|----------------------------------------------------------------------|
+| Ruby     | bundler                   | Gemfile.lock, gemspec                                                |
+| C#       | nuget                     | packages.lock.json, packages.config, .deps.json, *packages.props     |
```
Comment (aviat) [inline content/en/security/cloud_security_management/agentless_scanning/compatibility.md:35]: this should be .Net

**Comment by jinroh** [comment]
We could add a section on filesystems support. As of today we allow mounting:

- ext2
- ext3
- ext4
- xfs
- btrfs

**Comment by drichards-87** [comment]
> We could add a section on filesystems support. As of today we allow mounting:
> 
> * ext2
> * ext3
> * ext4
> * xfs
> * btrfs

I went ahead and created a separate Jira issue (DOCS-8364) to add this. That way, we can get the doc published sooner (hopefully later today before I go on PTO).

**Review by hestonhoffman — APPROVED** [review]
Small edit to frontmatter. LGTM

**Pull request merged**